### DISCUSSION TITLE
How do I safely upgrade to the latest version of terragrunt

### DISCUSSION THREAD

**Discussion by RazvanGherlea** [gruntwork-io/knowledge-base#852] [How do I]
Hello,

Terragrunt is serving over 150 AWS accounts. I am using github pipelines for planning and applying.
The plan is working as expected with the latest version `0.69.2`. Version `0.56.2` is currently used as the same time. 

Please can you share with me your experience ?

**Comment by yhakbar** [comment]
Hey @RazvanGherlea , do you mind sharing what problem you're experiencing while upgrading?

If you're trying to have a controlled rollout so that you can test changes, you can use a version manager like [mise](https://mise.jdx.dev/) to pin the version and bump the version via a commit so that you can easily progress through new versions in a controlled fashion.
